cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -672,51 +677,93 @@ public static ClientBuilder bluemix(String vcapServices) {
      * <ul>
      *     <li>The {@code vcapServices} is {@code null}.</li>
      *     <li>The {@code vcapServices} is not valid.</li>
+     *     <li>A service with the name matching "cloudantNoSQLDB" could not be found in
+     *     {@code vcapServices}.</li>
      *     <li>An instance with a name matching {@code instanceName} could not be found in
-     *     {@code vcapServices}.
+     *     {@code vcapServices}.</li>
      *     <li>The {@code instanceName} is {@code null} and multiple Cloudant service instances
      *     exist in {@code vcapServices}.</li>
      * </ul>
      */
     public static ClientBuilder bluemix(String vcapServices, String instanceName) {
-        Gson gson = new GsonBuilder().create();
+        return bluemix(vcapServices, "cloudantNoSQLDB", instanceName);
+    }
 
+    /**
+     * Sets Cloudant client credentials by inspecting a service information JSON string. This object
+     * takes the form of a {@code VCAP_SERVICES} environment variable which is a JSON object that
+     * contains information that you can use to interact with a service instance in Bluemix.
+     * <P>
+     * Note: Specifying an instance name is only required when multiple Cloudant service instances
+     * are present. If there is only a single Cloudant service instance then
```

Comment:
"If there is only a single Cloudant service instance then {@link #bluemix(String)} can also be used."
I don't think this is correct in the case where the single Cloudant service instance has a custom service name, as you'd need to provide `String, String`.

Reply:
Yeah, I guess if you've specified a custom `serviceName` and you don't want to specify an `instanceName` then you'd need `.bluemix(vcapServices, serviceName, null)`.

Reply:
I think I'll just remove the note entirely from this bluemix constructor.